Add Cloud (TSM) service notices and MQTT EOL notice (#5209)
influxdata/docs-v2: content/influxdb/cloud/reference/service-notices/native-collector-mqtt-eol.md

- [What are the expected next steps?](#what-are-the-expected-next-steps)
- [Will I lose any data already ingested?](#will-i-lose-any-data-already-ingested)
- [Where can I get more information on using Telegraf as a replacement for Native Collector - MQTT?](#where-can-i-get-more-information-on-using-telegraf-as-a-replacement-for-native-collector---mqtt)

### What is the Native Collector - MQTT Feature?

The Native Collector - MQTT feature was introduced in InfluxDB Cloud in 2022 as
a way to ingest data directly from MQTT sources into InfluxDB Cloud.
This feature has been removed and is no longer available to new customers or
existing customers who have not been using it.
It was only available to a limited number of existing customers who were
actively using it.

### Why is this feature being EOL'd?

The adoption of this feature was lower than anticipated given most customers
continued to use [Telegraf](/telegraf/v1/), InfluxData’s data collection agent,
which provides more features for MQTT ingestion than this feature did.
After a thorough assessment, we determined that our customers can have a similar

> (and even superior) experience with the

Reviewer: ```suggestion
(and even superior) experience using the
```

[MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/).

### What alternatives are available in light of this EOL announcement?

> We recommend using the [MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/)
> as an alternative to the Native Collector - MQTT feature.
> InfluxData’s very popular open source data collection agent with 300+ plugins,
> including MQTT.

Reviewer: ```suggestion
We recommend using [Telegraf](/telegraf/v1/) with the [MQTT Consumer Telegraf input plugin](https://www.influxdata.com/integration/mqtt-telegraf-consumer/)
as an alternative to the Native Collector - MQTT feature.

Telegraf is InfluxData’s popular open source data collection agent that comes with 300+ plugins for data input and output,
including MQTT.
```

### What are the expected next steps?

> The Native Collector - MQTT feature will continue to work and function until
> **April 30th, 2024**.  We will not be doing any new product development or
> enhancements to the feature in the meantime. After the EOL date, the feature
> will be decommissioned and any MQTT subscriptions that you have made will no
> longer receive and ingest data into InfluxDB.

Reviewer: ```suggestion
The Native Collector - MQTT feature will continue to work and function until
**April 30th, 2024**.
In the meantime, we won't release any new enhancements to the feature.
After the EOL date, we will decommission the feature, and any MQTT subscriptions that you have will no
longer receive and ingest data into InfluxDB.
```
